test/serializer/basic/PreventExtensions.js

```
@@ -0,0 +1,6 @@
+let obj = {x:1};
+Object.preventExtensions(obj);
+inspect = function() {
+  obj.y = 1;
```

Is this just silently ignored? Of course, I guess...